problems/CountNumSameDigits/src/main/java/Main.java

```diff
@@ -0,0 +1,52 @@
+import java.util.Scanner;
+
+// The task is to count numbers that have all the digits the same from the given range [a, b].
+public class Main {
+
+    public static void main (String[] args){
+
+        String inputRange = "88 99";
+
+        Scanner input = new Scanner(System.in);
+        int noLines = input.nextInt();
+        input.nextLine();
+        int [] arrResults = new int[noLines];
```

Comment:
Don't use `arr` as a prefix - it's generally not a good idea to have variable type in the name